src/main/resources/db/changelog/changes/01-create-book-table.yml

```
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-book-table
+      author: DanilKi
+      changes:
+        - createTable:
+            tableName: book
```

Table name should be plural
```suggestion
            tableName: books
```